google_maps/google_maps.py

```diff
@@ -1,58 +1,99 @@
 """ Google Maps XBlock """
 
 import pkg_resources
+from django.utils.translation import ugettext_lazy as _
 
 from xblock.core import XBlock
-from xblock.fields import Scope, Integer
+from xblock.fields import Scope, Integer, String
 from xblock.fragment import Fragment
+from xblockutils.resources import ResourceLoader
+
+loader = ResourceLoader(__name__)
 
 
 class GoogleMapsXBlock(XBlock):
     """
     XBlock providing a Google Map to a course.
     """
 
+    display_name = String(
+        display_name=_('Display Name'),
+        default=_('Google Maps XBlock'),
+        help=_('The display name for this component.'),
+        scope=Scope.settings
+    )
+
+    place_name = String(
+        display_name=_('Search for a place'),
+        default="",
+        scope=Scope.settings
+    )
+
+    iframe_width = Integer(
+        display_name=_('Map width:'),
```

Comment:
Move `:` out of the string. This is something that's presented on the template along with `display_name` for the purpose of indicating that the following input element contains the iframe_width value.
Add `:` in the template, after the display name.